### ISSUE TITLE
Add `commentPattern` option to `default-case` rule

### ISSUE THREAD

**Issue by alvov** [eslint/eslint#5803]
**What version of ESLint are you using?**
2.7.0
**What rule do you want to change?**
`default-case`
**What code should be flagged as incorrect with this change?**
```javascript
/*eslint default-case: ["error", { "commentPattern": "skip default case" }]*/
switch(foo) {
    case 0:
        a();
        break;
    case 1:
        b();
        break;
    // no default
}
```
**What happens when the rule is applied to this code now?**
Error occurs:
`Expected a default case.`

Originated in #5757, that issue is addressing configurable comment pattern for `default-case` rule. The point of proposed `commentPattern` option is to be able to override the default `no default` (no pun intended) comment pattern with a custom (regex) string.

**Comment by platinumazure** [comment]
As I've said in #5757, I am :+1: to this change because (among other reasons) it makes ESLint more usable by folks who don't speak English and who would prefer to write comments in another language.

**Comment by nzakas** [comment]
Sounds good. @alvov do you want to submit a PR?

**Comment by alvov** [comment]
@nzakas, yes, sure.
